java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/RequestPreparedStatement.java

```diff
@@ -258,6 +258,32 @@ public void setTimestamp(int i, Timestamp timestamp) throws SQLException {
         currentDatas.set(i - 1, ts);
     }
 
+    public void setRow(Map<String, Object> data) throws SQLException {
```

Comment:
Do not add methods that are not in jdbc. https://docs.oracle.com/javase/8/docs/api/java/sql/PreparedStatement.html